tensorboard/uploader/uploader_test.py

```diff
@@ -75,6 +75,25 @@ def test_create_experiment(self):
         eid = uploader.create_experiment()
         self.assertEqual(eid, "123")
 
+    def test_create_experiment_with_name_and_description(self):
```

Comment:
Please add tests for `update_metadata`, including requests sent and
failure handling.

Reply:
See UpdateExperimentMetadataTest below.  Is that sufficient?